dali/pipeline/operator/builtin/external_source.cc

```diff
@@ -86,11 +86,8 @@ and bandwidth.)code", false)
   .MakeInternal();
 
 DALI_SCHEMA(ExternalSource)
-  .DocStr(R"code(Allows externally provided data to be passed as an input to the pipeline,
-see :meth:`nvidia.dali.ops.ExternalSource.__init__`,
-see :meth:`nvidia.dali.fn.external_source`,
-:meth:`nvidia.dali.pipeline.Pipeline.feed_input` and
-:meth:`nvidia.dali.pipeline.Pipeline.iter_setup`.
+  .DocStr(R"code(Allows externally provided data to be passed as an input to the pipeline.
+
```

Comment:
Maybe you can still leave
```
:meth:`nvidia.dali.pipeline.Pipeline.feed_input` and	
:meth:`nvidia.dali.pipeline.Pipeline.iter_setup`.```

Reply:
Don't we want to stick to the new way of feeding external source? This would just confuse the user

Reply:
Good point.